README.md

```diff
@@ -145,7 +145,7 @@ Apprentices should memorize [**this**](Cheatsheet.md) within the first month.
 1. [Relational Databases](/databases/relational-databases.md)
 1. [Table Relationships and Join Queries by Treehouse](https://teamtreehouse.com/library/querying-relational-databases)
 1. [Adding Tests to your NodeJS project](/testing-and-tdd/integration-testing.md)
-1. [Database Migrations](/databases/database-migrations.md)
+1. [Database Migrations](/electives/databases/database-migrations.md)
```

Comment:
Please move all new electives to the electives section below.

Reply:
If anything has just become an elective, move it out of its present section and into the Elective section below.